packages/page-staking/src/Actions/Pool/useAccountInfoManualRewards.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Option } from '@polkadot/types';
+import type { PalletNominationPoolsPoolMember } from '@polkadot/types/lookup';
+import type { AccountInfo } from './types';
+
+import { useEffect, useState } from 'react';
+
+import usePoolInfo from '@polkadot/app-staking/Pools/usePoolInfo';
+import { createNamedHook, useApi, useCall, useIsMountedRef } from '@polkadot/react-hooks';
+import { PalletNominationPoolsBondedPoolInner, PalletNominationPoolsRewardPool } from '@polkadot/types/lookup';
+import { BN } from '@polkadot/util';
+
+const OPT_DEL = {
+  transform: (opt: Option<PalletNominationPoolsPoolMember>): PalletNominationPoolsPoolMember | null =>
+    opt.unwrapOr(null)
+};
+
+// as per polkadot-staking-dashboard code
+function calculatePayout (
+  bondedPool: PalletNominationPoolsBondedPoolInner,
+  rewardPool: PalletNominationPoolsRewardPool,
+  member: PalletNominationPoolsPoolMember,
+  rewardAccountBalance: BN
+) {
+  const rewardCounterBase = new BN(10).pow(new BN(18));
```

Comment:
Why 10^18?

Reply:
See this comment https://github.com/paritytech/substrate/blob/master/frame/nomination-pools/src/lib.rs#L438
Since original `pending_rewards` API call returns `BalanceOf<T>`, we need to scale counter by 10^18 manually to get balance